index.js

```diff
@@ -69,6 +78,16 @@ Shopify.prototype.request = function request(url, method, key, params) {
   return got(options).then(res => {
     const body = res.body;
 
+    const callLimitHeader = res.headers['x-shopify-shop-api-call-limit'];
+
+    if (typeof callLimitHeader === 'string') {
```

Comment:
As far as I know, it should be only undefined or a string. The typeof check is just a safety check in case anything goes awry.

I'll change it in the next squashed commit.